src/main/java/estafet/java14/Quarter.java

```diff
@@ -3,6 +3,11 @@
 import java.time.Month;
 
 public class Quarter {
+    /**
+     * Prints the quarter depending on the month
+     * @param month Month
+     * @return int
+     */
```

Comment:
In java 14  switch expression has been updated. Now you can have more than one value in case and you do not need break statements. For more info about switch expression check this site: https://docs.oracle.com/en/java/javase/14/language/switch-expressions.html